commit 4a2b1dbc605d2a9df3e40971fc2567731a93539c
Author: Jeroen Demeyer <jdemeyer@cage.ugent.be>
Date:   Wed Aug 1 13:45:00 2012 +0200

    Do not *build* the documentation when doing install-doc or install-docpdf.
    We must not build the documentation because that requires tex.  On the
    other hand, to have ? and ?? work within gp, we must install the .tex
    files (but not .dvi files).  So simply not doing install-doc doesn't
    work.

diff --git a/config/Makefile.SH b/config/Makefile.SH
index 6183942..b11a72a 100644
--- a/config/Makefile.SH
+++ b/config/Makefile.SH
@@ -404,7 +404,6 @@ install-cfg::
 
 install-doc::
 	-mkdir -p \$(BINDIR) \$(DOCDIR)
-	-cd $doc; \$(MAKE) all
 	-\$(INSTALL_PROGRAM) $doc/gphelp    \$(BINDIR)
 	-\$(INSTALL_DATA) $doc/Makefile     \$(DOCDIR)
 	-\$(INSTALL_DATA) $doc/translations \$(DOCDIR)
@@ -430,7 +429,6 @@ install-doc::
 
 install-docpdf::
 	-mkdir -p \$(BINDIR) \$(DOCDIR)
-	-cd $doc; \$(MAKE) docpdf
 	-\$(INSTALL_DATA) $doc/libpari.pdf  \$(DOCDIR)
 	-\$(INSTALL_DATA) $doc/users.pdf    \$(DOCDIR)
 	-\$(INSTALL_DATA) $doc/tutorial.pdf \$(DOCDIR)
